src/transaction/log_page_buffer.c

```diff
@@ -1269,15 +863,7 @@ logpb_set_dirty (THREAD_ENTRY * thread_p, LOG_PAGE * log_pgptr, int free_page)
     }
 #endif /* CUBRID_DEBUG */
 
-  START_EXCLUSIVE_ACCESS_LOG_PB (rv, thread_p);
-
   bufptr->dirty = true;
-  if (free_page == FREE)
```

Comment:
I have removed this parameter, because all it did was decreasing the fcnt, and we removed the fix count. I have updated the function, and now it doesn't have the free_page argument.